sql/parser/eval.go

```diff
@@ -259,6 +263,156 @@ func (d DString) String() string {
 	return StrVal(d).String()
 }
 
+// maxTime and minTime are not the theoretical max/min for the time.Time
```

Comment:
i'm late to this discussion, but what is the point of having maxTime and minTime? @petermattis originally suggested that IsMax and IsMin both return false, and that seems correct.